book/homework_1_solutions.ipynb

```diff
@@ -0,0 +1,3405 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.legend.Legend at 0x7f75ec615730>"
+      ]
+     },
+     "execution_count": 45,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create a plot that uses the colors assigned to the table of the Atomic Mass vs the Melting Point.\n",
+    "plt.figure()\n",
```

Comment:
You could also use the object oriented rather than the procedural interface for the two plots which don't have subplots.

```
fig, ax = plt.subplots()
ax.set_xlabel...
```

Reply:
This is probably a better idea. I have updated all the figures to use the object oriented api.

Reply:
I have updated all figures to use the object oriented api.